sysutils/pfSense-pkg-apcupsd/Makefile

```diff
@@ -0,0 +1,49 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-apcupsd
+PORTVERSION=	0.3.9
+CATEGORIES=	sysutils
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	coreteam@pfsense.org
+COMMENT=	pfSense package apcupsd
+
+LICENSE=	ESF
+
+RUN_DEPENDS=	${LOCALBASE}/sbin/apcupsd:sysutils/apcupsd \
+		/etc/inc/phpmailer/PHPMailerAutoload.php:mail/pfSense-pkg-mailreport
```

Comment:
Is mailreport mandatory? I don't like too much the idea of adding dependencies between additional packages